CONTRIBUTING.md

```diff
@@ -60,89 +60,74 @@ before submitting a PR.
 
 By contributing, you agree that your contributions will be licensed under its Apache License 2.0.
 
-# Building the solution
+## Preparing for a New Release
 
-The solution uses several open source software tools:
+The client uses generated stubs from upstream Qdrant proto definitions, which are downloaded from [qdrant/qdrant](https://github.com/qdrant/qdrant/tree/master/lib/api/src/grpc/proto).
 
-## Docker
+The generated files do not form part of the checked in source code. Instead, they are generated
+and emitted into the `src/Qdrant.Client/obj/Release`, and included in compilation.
 
-Qdrant docker image is used to run integration tests. Be sure to 
-[install docker](https://docs.docker.com/engine/install/) and have it running when running tests.
+### Pre-requisites
 
-## Bullseye
+Ensure the following are installed and available in the `PATH`.
 
-[Bullseye](https://github.com/adamralph/bullseye) is used as the build automation system for the solution.
-To get started after cloning the solution, it's best to run the build script in the root
+- [Dotnet 6.0.x](https://dotnet.microsoft.com/download/dotnet/6.0)
```

Comment:
```suggestion
- [Dotnet 8.0.x](https://dotnet.microsoft.com/download/dotnet/8.0)
```

The tests use .NET 8, and is backwards compatible with .NET 6. The global.json file in the root dictates the SDK version needed for the solution.